fitbenchmarking/fitbenchmarking
# Extend algorithm type

#### Description of Work

Fixes #782 

#### Testing Instructions

1. Check that new algorithm type options make sense, should any be added/removed?
2. Run fitbenchmarking with various algorithm type options and ensure that algorithm type is displayed as a tooltip when hovering over minimizer name in results tables

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
